mobile-app/App.tsx

```diff
@@ -1,19 +1,39 @@
-import { StatusBar } from 'expo-status-bar';
-import { StyleSheet, Text, View } from 'react-native';
+import * as React from 'react';
+import { StyleSheet, View, ScrollView } from 'react-native';
+import Board from './components/Board';
+import { GameState } from '../web-app/src/common/gameState';
+
+const gamestate: GameState = {
+  turn: 'white',
+  winner: null,
+  winningPath: null,
+  board: [
+    [{ value: 'empty' }, { value: 'empty' }, { value: 'empty' }, { value: 'empty' }, { value: 'empty' }, { value: 'empty' }, { value: 'empty' }, { value: 'empty' }, { value: 'empty' }],
```

Comment:
'empty' should be a constant, same for both player values

Reply:
We are using an interface defined previously : 

```
export interface Cell {
  value: 'empty' | 'black' | 'white';
}
```

So we already have some protection with type checking and autocompletion, so does consts values are still needed here ?

Reply:
Good point for the type checking 👍 
I suggest you to use enum : https://www.typescriptlang.org/docs/handbook/enums.html

Reply:
You're right, after searching on google, majority of articles talks about using union types over enum :)
i'm agree with you 👍